developers/weaviate/search/similarity.md

```diff
@@ -14,29 +14,25 @@ import TSCode from '!!raw-loader!/_includes/code/howto/search.similarity.ts';
 
 import ClassToCollectionNote from '/_includes/class-to-collection-transition-note.mdx' ;
 
-<ClassToCollectionNote /> 
+<ClassToCollectionNote />
 
-This page has examples of vector search. 
+This page has examples of vector search.
 
 <details>
   <summary>Additional information</summary>
 
   Vector search is a similarity based search. The vector search operators look for objects with vector representations that are similar to the query's vector representation.
-     
+
     - For search concepts, see [Search](/developers/weaviate/concepts/search).
     - For image search, see [Image search](/developers/weaviate/search/image).
     - For tutorials, see [Queries](/developers/academy/zero_to_mvp/queries_1).
     - For search using the GraphQL API, see [GraphQL API](/developers/weaviate/api/graphql).
 
 </details>
 
-## Search with unvectorized input
-
-If you use unvectorized inputs to search with, the collection must have a [vectorizer](../modules/retriever-vectorizer-modules/index.md) configured.
+## Search with text
 
-- For text object, use [`nearText`](../api/graphql/search-operators.md#neartext).
-- For image objects, use [`nearImage`](./image.md). 
-- For mixed text and images, use [CLIP](../modules/retriever-vectorizer-modules/multi2vec-clip.md).
+You can use use [`nearText`](../api/graphql/search-operators.md#neartext) to find objects with the nearest vector to an input.
```

Comment:
The example is just below.
Do we need a link to the reference page here? (this a question, not a directive)

Reply:
let's keep it then